src/main/java/org/tools4j/hoverraft/message/direct/DirectCommandMessage.java

```diff
@@ -96,4 +96,11 @@ public void unwrap() {
         command.unwrap();
         super.unwrap();
     }
+
+
+    @Override
+    public void copyFrom(final CommandMessage commandMessage) {
```

Comment:
don-t understand. can't override with subtype (you can OVERLOAD but not OVERRIDE)